CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # changelog
 
+## 6.4.3 - 15 Feb 2017
+
+* Create a wrapper for `listBuildTargets` in `Client.js`.
+* Marked `compileCode`, `signalDevice`, `listDevices` and `listBuildTargets` as deprecated. Those methods will be removed in 6.5
```

Comment:
They will be replaced with collection/model pattern once we decide how exactly it should look like. I'm marking them now so anyone trying to use them will be warned that this functionality will change soon.